README.md

```diff
@@ -0,0 +1,3 @@
+# conferences-multi-module-heroku
+
+HEY THERE FROM NIR PROGRAMMER COLLABORATOR FROM UBUNTU!!
```

Comment:
But I need first the mor "!!!!!" in the README.md.
Chears @nir-devel  - The owner of this repo, from Windows!

Reply:
OK TAKE THIS!!!!